pkg/metaclient/internal/interface.go

```diff
@@ -0,0 +1,15 @@
+package internal
+
+import (
+	"context"
+
+	"github.com/hanfei1991/microcosm/pkg/metaclient"
+)
+
+type KVEx interface {
```

Comment:
What does `Ex` mean?

Reply:
Extend the KV interface with Do method to implement the intermidate layer easier